db/migrate/20191009004932_create_bulk_downloads_table.rb

```diff
@@ -0,0 +1,18 @@
+class CreateBulkDownloadsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bulk_downloads do |t|
+      t.text "params_json", comment: "JSON of the params for this bulk download"
+      t.string "download_type", comment: "The type of bulk download"
+      t.string "status", comment: "The current status of the download, e.g. waiting, running, error, success"
+      t.string "error_message", comment: "An error message to display to the user."
```

Comment:
it seems like every column except this one should be not null. if so, please add `null: false` to the definitions.